xml/http_servers.xml

```diff
@@ -3283,4 +3283,12 @@
     <param pos="0" name="service.product" value="Symantec Endpoint Protection Manager"/>
     <param pos="0" name="service.family" value="Symantec Endpoint Protection Manager"/>
   </fingerprint>
+  <fingerprint pattern="^Intel\(R\) Active Management Technology\s(\d+\.\d+\.\d+\.\d+|\d+\.\d+\.\d+|\d+\.\d+)">
```

Comment:
There are older versions that are a subset of AMT called Standard Manageability that has nearly the same fingerprint and is affected by much of the same security issues as of late.   I'll send you a PR